src/user.rs

```diff
@@ -155,6 +158,7 @@ impl User<Disconnected> {
                     client: self.client.clone(),
                     tx: self.tx.clone(),
                     state: Registered {},
+                    friendships: vec![],
```

Comment:
Shouldn't we use the `self.friendships` here instead of a new vector?

Reply:
Yes, though they will be empty since it's a new user

Reply:
For now, but I would prefer to change it since it's a no brainer and might be useful in the future.

Reply:
Of course, it was changed for that same reason